grails-app/services/org/pih/warehouse/data/ProductAssociationDataService.groovy

```diff
@@ -24,16 +25,30 @@ class ProductAssociationDataService {
         log.info "Validate data " + command.filename
 
         command.data.eachWithIndex { params, index ->
+            DateFormat dateFormat = new SimpleDateFormat("yyyy-mm-dd hh:mm:ss")
+            if (params['dateCreated']) params['dateCreated'] = dateFormat.parse(params['dateCreated'])
+            if (params['lastUpdated']) params['lastUpdated'] = dateFormat.parse(params['lastUpdated'])
```

Comment:
Why you are importing these datetimes? Why it is not handled by the gorm as an auto timestamp? Is there a case where this makes sense to import these values?

Reply:
I thought about that users can set these dates themselves instead of it beeing automatically assigned, just like on manual create.

But now that I think abut it it doe snot make any sens for users to have any influence on setting `dateCreated` `lastUpdated` values, it should happen automatically, so I will remove it and ignore these fields

Reply:
additionally, even though on manual creation for product association we have date fields for **Date Created** and **Las updated** they don't work